adminpage/sport/admin/utils.py

```diff
@@ -128,3 +129,45 @@ def has_output(self):
             return self.no_output
 
     return CacheRelatedFieldListFilter
+
+
+def has_free_places_filter():
+    class Wrapper(admin.SimpleListFilter):
+        title = 'free places'
+        parameter_name = 'has_free_places'
+
+        def lookups(self, request, model_admin):
+            return (
+                ('1', _('Has free places')),
+                ('0', _('Group is full')),
+            )
+
+        def queryset(self, request, queryset):
+            if self.value() == '1':
+                return queryset.filter(capacity__gt=F('enroll_count'))
+            if self.value() == '0':
+                return queryset.filter(capacity__lte=F('enroll_count'))
+            return queryset
+
+    return Wrapper
+
+
+def has_enrolled_filter():
```

Comment:
Leave a comment that you should manually add `has_enrolled` field (since it isn't a field of any model)